dev/breeze/src/airflow_breeze/prepare_providers/provider_documentation.py

```diff
@@ -715,13 +716,51 @@ def _update_commits_rst(
     )
 
 
+def _is_test_only_changes(pr_number: str, github_token: str | None = None) -> bool:
```

Comment:
As discussed on slack - adding the neef for GH token here is likely not a good idea for the interactive command. We can query local git repo without extra interaction with GH because RM already has all the commits locally - and we have all data in "Changes" (notably commit hash).

```
class Change(NamedTuple):
    full_hash: str
    short_hash: str
    date: str
    message: str
    message_without_backticks: str
    pr: int | None
```

Either by pygit api or even easier by `git command`:

```
git diff 4dfc907f6bdc86a4c9bb941f61accebdc5333b13 4dfc907f6bdc86a4c9bb941f61accebdc5333b13^ --name-only 
providers/fab/tests/conftest.py
providers/fab/tests/unit/fab/www/views/test_views_custom_user_views.py
```

No authentication needed.

Reply:
Good call. I will try along these lines, wasnt so sure about wheteher we pull it locally, turns out we do!